test/test.js

```diff
@@ -0,0 +1,159 @@
+var assert = require('chai').assert,
```

Comment:
Also can you add a test for the `verify` method? You don't actually ahve to handle an IPN, but just two tests: a valid signature and an invalid signature(you can hand create the data to be verified if you want)